packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -61,22 +62,27 @@ export const App = ({ flowQueryParams }: AppProps) => {
   }
 
   return (
-    <AppLayout>
-      <Head />
-      <Router basepath={HomePath}>
-        <ScrollToTop default>
-          <PageSettings path="/" />
-          <FlowContainer path="/avatar/change" title="Profile picture" />
-          <PageDisplayName path="/display_name" />
-          <PageChangePassword path="/change_password" />
-          <PageRecoveryKeyAdd path="/account_recovery" />
-          <PageSecondaryEmailAdd path="/emails" />
-          <PageSecondaryEmailVerify path="/emails/verify" />
-          <PageTwoStepAuthentication path="/two_step_authentication" />
-          <Page2faReplaceRecoveryCodes path="/two_step_authentication/replace_codes" />
-        </ScrollToTop>
-      </Router>
-    </AppLayout>
+    <AppLocalizationProvider
+      baseDir="/beta/settings/locales"
```

Comment:
Best to use https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/constants/index.tsx#L5 instead of a literal here.